WordPress/src/main/java/org/wordpress/android/datasets/AsyncIoTaskExecutor.kt

```diff
@@ -26,7 +28,7 @@ object AsyncTaskHandler {
         }
     }
 
-    interface AsyncTaskCallback<T> {
+    interface IoTaskResultCallback<T> {
```

Comment:
If going with the suggestion of keeping the class generic in relation to Dispatchers, this interface name change could be reverted.